WordPress/Classes/ViewRelated/Blog/WPStyleGuide+Sharing.swift

```diff
@@ -16,8 +17,7 @@ extension WPStyleGuide
         let horizontalPadding = 8.0;
         let imageView = UIImageView(frame:CGRect(x: 0, y: 0, width: imageSize + horizontalPadding, height: imageSize))
 
-        let noticeImage = UIImage(named: "gridicons-notice")
-        imageView.image = noticeImage?.imageWithRenderingMode(.AlwaysTemplate)
+        imageView.image = UIImage(named: "sharing-notice")
```

Comment:
@aerych I set the image as 'template' in the asset catalog: https://github.com/wordpress-mobile/WordPress-iOS/pull/5096/commits/591fc330d18aa586cb134e75f62303aa12e9dabd#diff-7eef83a33166c97c4ab94e829cd5f91bR13